tutorials/models/1_gnn/4_rgcn.py

```diff
@@ -362,4 +362,4 @@ def forward(self, g):
 # The implementation is similar to that presented here, but with an extra DistMult layer
 # stacked on top of the R-GCN layers. You can find the complete
 # implementation of link prediction with R-GCN in our `Github Python code example
-#  <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link_predict.py>`_.
+# <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link.py>`_.

```

Comment:
have you checked whether this is displayed well? I don't think so. What I confirmed is `example <https....` works as expected.

Reply:
No, I did not build the docs, I thought this would be an easy fix. I'm having issues building the library right now. I will allocate more time for this soon.